docs/releases/1.22-NOTES.md

```diff
@@ -6,6 +6,20 @@ This is a document to gather the release notes prior to the release.
 
 # Significant changes
 
+## Instance metadata service version 2
+
+ On AWS, kOps will enable [Instance Metadata Service Version 2](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html) by default with a max-hop-limit of 1 on new clusters that use Kubernetes 1.22. This means that any container running in the cluster will be unable to connect to the instance metadata _unless_ the container is running with `hostNetworking: true`. This will increase security by default, but may break some types of workloads. In order to revert to old behavior, add the following to the InstanceGroup:
+
+```
+spec:
+  instanceMetadata:
+    httpPutResponseHopLimit: 1
+    httpTokens: optional
+```
+
+This change does not affect control plane nodes.
```

Comment:
I think this change will affect control plane nodes also.

Reply:
Changed the wording to it is more clear.

Reply:
The problem is not the wording but the intention.
As @rifelpet was saying above, conclusion from office hours was:
> **all node roles**, token requred, hop limit 1

Reply:
We  cannot implement this on master roles though. Not until IRSA is used for all components and enabled by default.